### ISSUE TITLE
Image disappears from the note preview if a width and/or height is specified

### ISSUE THREAD

**Issue by pmac** [mozilla/nucleus#783]
When using an `<img>` tag, Nucleus' preview will show the image when just the src and alt are specified, but once width or height are added, the image disappears. The images do load correctly on Bedrock, however.

imported from https://bugzilla.mozilla.org/show_bug.cgi?id=1850060

**Comment by pmac** [comment]
I would think so, but it happens in the client-side auto-updating preview and I'm not sure Python is involved, but they might have something similar in the JS.

**Comment by stevejalim** [comment]
OK so playing about with this to reproduce, I can make it work with no code changes to Nucleus. 

The niggle is rooted in the pagedown library, which we use off-the-shelf, so I tried to reproduce the same issue using it on StackOverflow's profile editor and discovered the same behaviour. 

Fiddling around there I discovered that the order of the attributes seems to matter here so this works fine

```<img src="https://placekitten.com/200/200" width="20" alt="test image of some cats">```

But this does not
```<img src="https://placekitten.com/200/200" alt="test image of some cats" width="20">```

I've tried the same in Nucleus, locally, and can confirm the attribute order 'fix' also works for us too - see [this screen recording](https://drive.google.com/file/d/11BuyvYKbEhCxGD_O6q1LbqcxG5j3h6BC/view?usp=drive_link) 

@rvandermeulen is this OK as a workaround for you?
